model-test/src/test/java/hu/blackbelt/judo/test/specification/TCS018.java

```diff
@@ -0,0 +1,56 @@
+package hu.blackbelt.judo.test.specification;
+
+public interface TCS018 {
```

Comment:
I don't understand why we need a separate interface and implementation. Could there be another implementations of the same test? If not then the "specification" below can be moved to the actual implementation.

If the idea was to separate the specification and the test implementation to decrease lines of code in a file,  I don't think the use of interfaces is proper solution.